src/pkg/packaging/rpm/dotnet-hostfxr-rpm_config.json

```diff
@@ -0,0 +1,43 @@
+{
+  "maintainer_name": "Microsoft",
+  "maintainer_email": "dotnetcore@microsoft.com",
+  
+  "package_name": "%HOSTFXR_RPM_PACKAGE_NAME%",
+  "install_root": "/usr/share/dotnet",
```

Comment:
@ellismg Using openjdk as an example: it should be okay to put everything under `/usr/lib64/dotnet`.